src/popup/components/ConnectWalletForm.tsx

```diff
@@ -153,23 +159,19 @@ export const ConnectWalletForm = ({
       }
       saveValue('amount', error ? value : amountValue);
     },
-    [saveValue, currencySymbol, t],
+    [saveValue, currencySymbol, toErrorInfo],
   );
 
-  const handleSubmit = async (ev: React.FormEvent<HTMLFormElement>) => {
-    ev.preventDefault();
-    if (!walletAddressInfo) {
-      setErrors((_) => ({ ..._, walletAddressUrl: 'Not fetched yet?!' }));
-      return;
-    }
+  const handleSubmit = async (ev?: React.FormEvent<HTMLFormElement>) => {
+    ev?.preventDefault();
```

Comment:
Why is the event optional now?

Reply:
Because now we manually trigger the submit (without the form also), like on "Accept consent" or "retry".

Reply:
I realize it's not used in this PR, but only in https://github.com/interledger/web-monetization-extension/pull/637. Will remove for this PR.